CHANGELOG.md

```diff
@@ -0,0 +1,140 @@
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-25'>AUTH-25</a>] -         Design user token that will be used to represent the logged in user
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-26'>AUTH-26</a>] -         A proof of concept that shows how to consume an identity token across several components of the platforms
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-41'>AUTH-41</a>] -         A proof of concept that shows how to consume an identity token across several components of the platforms
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-46'>AUTH-46</a>] -         Investigate how AWS Cognito could be used directly
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-64'>AUTH-64</a>] -         Integrate code into ACS to implement the use of the internal auth token
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-67'>AUTH-67</a>] -         Ensure Keycloak Realm is Configurable at Deploy Time
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-68'>AUTH-68</a>] -         Use the Official Keycloak Helm Chart
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-69'>AUTH-69</a>] -         Update Hello World to demonstrate SSO
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-85'>AUTH-85</a>] -         Move SSO token implementation back to Alfresco Community
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-94'>AUTH-94</a>] -         Identity Service (Keycloak) Persistence
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-97'>AUTH-97</a>] -         CLONE - Ensure Keycloak Realm is Configurable at Deploy Time
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-106'>AUTH-106</a>] -         Automated testing of SSO for APS and ACS
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-118'>AUTH-118</a>] -         Update infrastructure to use the Alfresco-identity service
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-119'>AUTH-119</a>] -         Automate regression testing for SAML testing
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-121'>AUTH-121</a>] -         Enable Automated Testing for SSO
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-126'>AUTH-126</a>] -         DBP Postman tests refactoring
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-142'>AUTH-142</a>] -         Update Identity Service description
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-143'>AUTH-143</a>] -         Separate Test Cases for Authorization
+</li>
+<li>[<a href='https://issues.alfresco.com/jira/browse/AUTH-146'>AUTH-146</a>] -         Identity Service implicit flow regression testing
```

Comment:
This work is going to be delivered in AUTH-215. I'd remove this ticket too.

Reply:
I removed AUTH-215